### PR TITLE
remove unnecessary info from android and ios

### PR DESCRIPTION
**Pull request by archlichking**
platform --> Linux if osName === 'Android'
platform --> OS X 10.10 if osName === 'iOS' 

and remove resolutions for both platforms

### CODE REVIEW

File: src/browsers.js [hunk 1]
```diff
@@ -331,11 +331,25 @@ var SauceBrowsers = {
...
+          case 'iOS':
+            // iOS is not a platform name, use OS X 10.10 instead
+            result.platform = 'OS X 10.10';
+            break;
+          case 'Android':
+            // Android is not a platform name, use Linux instead
+            result.platform = 'Linux';
+            break;
+          default:
+            result.platform = osName;
```
Comment (geekdave) [inline src/browsers.js]: > only append resolution to none Android or iOS configuration

might be clearer as `only append resolution to desktop clients`

**Comment by geekdave** [comment]
Failing CI, and a minor suggestion.  Let's wait for @Maciek416 's review on this before merging.

**Comment by archlichking** [comment]
There is an issue with this PR, I'm checking.

**Comment by archlichking** [comment]
Saucelabs cached file has been updated for unit test.

File: test/standard-capabilities.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+    var target = result[0];
+    expect(target.desiredCapabilities).to.have.property("browserName", "android");
+    expect(target.desiredCapabilities).to.have.property("platform", "Linux");
+    expect(target.desiredCapabilities).to.have.property("version", "4.1");
+    expect(target.desiredCapabilities).to.have.property("deviceName", "Samsung Galaxy S3 Emulator");
+  });
+
+  it.skip("standardize saucelabs capabilities of android emulator using appium", function() {
+    // TODO: android with appium is tricky, we need to think about how to test
```
Comment (Maciek416) [inline test/standard-capabilities.js]: Why is this tricky?

Comment (archlichking) [inline test/standard-capabilities.js]: android with appium has web, hybrid and native app test configs and they are all different, this is a case by case scenario.

**Pull request merged**